DEPS

```diff
@@ -352,7 +352,7 @@ deps = {
    Var('dart_git') + '/yaml.git@b4c4411631bda556ce9a45af1ab0eecaf9f3ac53',
 
   'src/third_party/dart/third_party/pkg_tested/dart_style':
-   Var('dart_git') + '/dart_style.git@0fe592042eda5d9eea6fc240a8ee0fe531bb0794',
+   Var('dart_git') + '/dart_style.git@09d9dff90d9a2e0793ad2f795f36c2777f720eda0',
```

Comment:
looks like another bad copy paste? you have an extra leading 0

Reply:
Nice catch! Done